contracts/jobs/libraries/TranscodeJobs.sol

```diff
@@ -179,13 +179,14 @@ library TranscodeJobs {
      * @param _proof Merkle proof for the signed transcode claim
      * @param _verificationRate % of segments to be verified
      */
-    function validateTranscoderClaim(Jobs storage self,
+    function validateTranscodeClaim(Jobs storage self,
                                      uint256 _jobId,
                                      uint256 _segmentSequenceNumber,
-                                     bytes32 _dataHash, bytes32 _transcodedDataHash,
+                                     string _dataHash,
+                                     string _transcodedDataHash,
                                      bytes _broadcasterSig,
                                      bytes _proof,
-                                     uint64 _verificationRate) constant returns (bool) {
+                                     uint64 _verificationRate) internal constant returns (bool) {
```

Comment:
Due to the type change of `_dataHash` and `_transcodedDataHash` from `bytes32` to `string` I ran into a `Stack too deep, try removing local variables` compiler error. Local variables including function parameters and return parameters are placed on the stack and the EVM can only reach a specific depth for the stack - the current compiler defined max is 16. I need to investigate this further, but an initial guess as to why the compiler complained when this library function was public is that since library functions are invoked using `delegatecall`, all the variables are copied which perhaps leads to them being placed on the stack. When the library function is internal, library code is copied into the caller function and variables are referenced rather than copied.